RSA/rsa-securid/ingest/parser.yml

```
@@ -0,0 +1,102 @@
+name: rsa-securid
+pipeline:
+  - name: parsed_event
+    external:
+      name: dsv.parse-dsv
+      properties:
+        input_field: original.message
+        output_field: message
+        columnnames:
+          - TIME
+          - MS_TIME
+          - OBSERVER_HOSTNAME
+          - RSA_SECUREID_CLASS
+          - LOG_LEVEL
+          - ???
+          - OBSERVER_SERIAL_NUMBER
+          - DESTINATION_IP
+          - SOURCE_IP
+          - RSA_SECUREID_ACTION_NAME
+          - EVENT_CODE
+          - RSA_SECUREID_EVENT_OUTCOME
+          - EVENT_REASON
+          - RSA_SECUREID_SESSION_ID
+          - USER_ID
+          - RSA_SECUREID_SOURCE_ID
+          - RSA_SECUREID_DOMAIN_ID
+          - USER_NAME
+          - USER_FIRSTNAME
+          - USER_LASTNAME
+          - RSA_SECUREID_AGENT_IP
+          - AGENT_NAME
+          - RSA_SECUREID_POLICY_METHOD_ID
+          - RSA_SECUREID_POLICY_METHOD_NAME
+          - RSA_SECUREID_POLICY_ID
+          - RSA_SECUREID_POLICY_EXPRESSION
+          - RSA_SECUREID_OBJECTS_TYPE
+          - RSA_SECUREID_OBJECTS_ID
+          - RSA_SECUREID_OBJECTS_SOURCE_ID
+          - RSA_SECUREID_OBJECTS_SECURITY_ID
+          - RSA_SECUREID_OBJECTS_NAME
+          - RSA_SECUREID_OBJECTS_TYPE
+          - ARG7
+          - ARG8
+          - ARG9
+          - ARG10
+        delimiter: ","
+
+  - name: parsed_description
+    external:
+      name: grok.match
+      properties:
+        input_field: original.message
+        output_field: message
+        pattern: '%{GREEDYDATA:rsaadmin_title}: %{GREEDYDATA:tty}; %{GREEDYDATA:pwd} ; %{GREEDYDATA:user} ; %{GREEDYDATA:command}|%{GREEDYDATA:reason}'
```

Extracting rsaadmin is not required. Add prefix to each capture.
```suggestion
        pattern: 'rsaadmin: TTY=%{GREEDYDATA:tty}; PWD=%{GREEDYDATA:pwd} ; USER=%{GREEDYDATA:user} ; COMMAND=%{GREEDYDATA:command}|%{GREEDYDATA:reason}'
```